### PR TITLE
Rel 539853 | Updated HTTP requests to include response if an exception is thrown

### PR DESCRIPTION
**Pull request by jake-martin**
Response will be printed to the console (via the thrown exception) if any HTTP requests are not successful.

### CODE REVIEW

**Review by aarongilbert123 — APPROVED** [review]

File: Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb [hunk 1]
```diff
@@ -5,12 +5,24 @@
...
-    
-    New-Workspace -RelativityInstanceName #{node['windows']['new_computer_name']} -RelativityAdminUserName #{node['relativity']['admin']['login']} -RelativityAdminPassword #{node['relativity']['admin']['password']} -SqlAdminUserName #{node['sql']['user']['eddsdbo']['login']} -SqlAdminPassword #{node['sql']['user']['sa']['password']} -WorkspaceName '#{node['sample_data_grid_workspace_name']}' -EnableDataGrid true
+    New-Workspace -RelativityInstanceName #{node['windows']['new_computer_name']} -RelativityAdminUserName #{node['relativity']['admin']['login']} -RelativityAdminPassword #{node['relativity']['admin']['password']} -SqlAdminUserName #{node['sql']['user']['eddsdbo']['login']} -SqlAdminPassword #{node['sql']['user']['sa']['password']} -WorkspaceName '#{node['sample_workspace_name']}' -EnableDataGrid false -ErrorAction Stop
```
Comment (calimetikcura) [inline Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb:23]: @jake-martin One more thing I noticed was this PowerShell module didn't stop when it encountered an exception and failed down the line. So I added "-ErrorAction Stop" to force the stop when there is an error.
This is one thing I forgot to mention to you. 

- [ ] Can you please add "-ErrorAction Stop" for rest of the PS modules calls?

**Review by calimetikcura — CHANGES_REQUESTED** [review]
Added one more task to update.

File: Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb (hunk 1 above)
Comment (jake-martin) [reply Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb:23]: @calimetikcura Done. I re-ran the script to ensure there were no syntax errors.

**Review by calimetikcura — APPROVED** [review]

File: Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb (hunk 1 above)
Comment (calimetikcura) [reply Chef/Cookbooks/Relativity/recipes/post_relativity_powershell_module_add_workspaces.rb:23]: Awesome! Good job @jake-martin for running the scripts for confirmation!

**Review by TommyBigNose — APPROVED** [review]

**Pull request merged**